### PR TITLE
chore: add link to spectral video

### PR DESCRIPTION
**Pull request by ssspear**
* Add ability to map over multiple call to actions

### CODE REVIEW

File: src/components/Hero/index.tsx [hunk 1]
```diff
@@ -176,16 +178,28 @@ export const Hero: React.FunctionComponent<IHero> = ({
...
-            <CallToAction
-              className={cn('pb-16 md:pb-4', {
-                'mx-auto': aligned === 'center',
-                'ml-auto': aligned === 'right',
-                'mr-auto': aligned === 'left',
-              })}
-              {...cta}
-            />
-          )}
+          {cta &&
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: I don't think we need to support both `cta` and `ctas`. Lets just support `ctas` and if you just want one then you can pass it in as an array with one item

File: src/components/Hero/index.tsx [hunk 2]
```diff
@@ -176,16 +178,28 @@ export const Hero: React.FunctionComponent<IHero> = ({
...
+                  'mr-auto': aligned === 'left',
+                })}
+                {...cta}
+              />
+            ) // only one cta button
+          }
+
+          {ctas &&
+            !cta && (
+              <div className="flex justify-center md:flex-col">
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: I think we might want that same margin logic from above. Have you tried adding a CTA on a page that is left aligned or right aligned?

```
{
  'mx-auto': aligned === 'center',
  'ml-auto': aligned === 'right',
  'mr-auto': aligned === 'left',
}
```

File: src/components/Hero/index.tsx [hunk 3]
```diff
@@ -42,6 +42,7 @@ export interface IHero {
   contentBgOverlay?: string;
   aligned?: 'center' | 'right' | 'left';
   cta?: ICallToAction;
+  ctas?: ICallToAction[];
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: We will also need to update the Netlify CMS config file for this component to include this new `ctas` option: https://github.com/stoplightio/stoplight.io/blob/develop/src/components/Hero/config.js#L36

This config file tells Netlify CMS how to render the input form in the admin panel. Check out their docs on how to configure an array: https://www.netlifycms.org/docs/widgets/

File: src/sections/ImageCallout/index.tsx [hunk 4]
```diff
@@ -28,6 +29,7 @@ export const ImageCallout: React.FunctionComponent<IImageCallout> = ({ title, ct
           />
 
           {cta && <CallToAction {...cta} className="md:mx-auto" />}
+          {ctas && ctas.map((action, i) => <CallToAction key={i} className="pb-16 px-4 md:pb-4" {...action} />)}
```
Comment (lottamus) [inline src/sections/ImageCallout/index.tsx]: Same as above, I think we should just support an array of `ctas`. We probably want to wrap these in a flexbox so they're next to each other too?

File: src/components/Hero/index.tsx (hunk 1 above)
Comment (ssspear) [reply src/components/Hero/index.tsx]: Sounds good. Note: To my knowledge, all of the other pages are passing in `cta`, so my thought was I would need to update the other yaml files to `ctas`. Is that correct?

**Comment by lottamus** [comment]
Is the goal of this PR to add a link to the video or actually show the video on our site?

Ideally we allow the CTA to have a `type` property which can be a `link` or `video`. If the action is a `video`, clicking it would open a modal that has the video embedded inside of it.

Since we don't have blueprint in here yet you could probably use some open source video player and modal such as https://github.com/CookPete/react-player and https://github.com/pradel/react-responsive-modal

File: src/components/Hero/index.tsx (hunk 1 above)
Comment (lottamus) [reply src/components/Hero/index.tsx]: Yeah only for the "hero" config though: https://github.com/stoplightio/stoplight.io/blob/develop/netlify/landings/lp-design.yaml#L9

**Comment by tbarn** [comment]
The initial plan was just to have a link that could be used for other things than just video depending on the landing page. It doesn't really need to be in-page. Getting people out to our now growing YouTube channel wouldn't be a bad thing.

**Comment by tbarn** [comment]
@lottamus can we merge this? we'd like it for Prism landing page too.

**Comment by lottamus** [comment]
> can we merge this?

@tbarn yes I'll make sure it gets merged in today. I will update the Prism page when it is in too

**Pull request merged**